src/main/scala/commonist/util/OaiPmh2.scala

```diff
@@ -0,0 +1,122 @@
+package commonist.util
+
+import commonist.data._
+
+import scala.xml._
+
+import scutil.log._
+
+/** 
+ * Parse and extract image metadata from OAI-PMH 2.0 files.
+ * See www.openarchives.org/OAI/openarchivesprotocol.html
+ */
+class OaiPmh2(doc:Elem,props:Map[String,String]) extends Logging {
```

Comment:
i don't thinks a class is ideal here - this is just a couple of functions. but if it has to be one, then it should be final.

Reply:
i'd probably  just use an object as a namespace for a bunch of functions - you'd need to pass around the doc and the props maybe, but at least that would make it clear whee exactly something is used.